inbm-lib/inbm_common_lib/utility.py

```diff
@@ -190,7 +190,26 @@ def validate_file_type(path: List[str]) -> None:
     for tarball in tarball_list:
         with tarfile.open(tarball) as tar:
             logger.debug("Extract {0}.".format(tarball))
-            tar.extractall(path=TEMP_EXT_FOLDER)
+            def is_within_directory(directory, target):
+                
+                abs_directory = os.path.abspath(directory)
+                abs_target = os.path.abspath(target)
+            
+                prefix = os.path.commonprefix([abs_directory, abs_target])
+                
+                return prefix == abs_directory
+            
+            def safe_extract(tar, path=".", members=None, *, numeric_owner=False):
+            
+                for member in tar.getmembers():
+                    member_path = os.path.join(path, member.name)
+                    if not is_within_directory(path, member_path):
+                        raise Exception("Attempted Path Traversal in Tar File")
```

Comment:
Change to use correct exception type for inbm-lib

Reply:
@nmgaston understood. I wanted to make sure I flagged the issue and didn't forget about it later.